spec/factories.rb

```diff
@@ -5,9 +5,9 @@
   end
 
   factory :rover do
-    name { "Curiosity" }
```

Comment:
This is likely going to cause failures in a bunch of existing tests. Rather than change these fields in this factory, it would make more sense to simply set them to what you want in your `PerseveranceScraper` test, like `create(:rover, name: "Perseverance", landing_date: Date.new(2021, 2, 18), launch_date: Date.new(2020,  7, 30))` on line 4 of `spec/models/perseverance_scraper_spec.rb`.